e2e/README.md

```diff
@@ -0,0 +1,31 @@
+# E2E tests
+
+
+## Directory structure
+
+- `test-app-v10`: example app using Angular v10 with basic setup with `jest` and `jest-preset-angular`.
+- `test-app-v9`: example app using Angular v9 with basic setup with `jest` and `jest-preset-angular`.
+
+
+## Running tests
+
+You run the tests on example apps with `yarn test:e2e` (or `npm run test:e2e)`. In `scripts/lib/paths.js`,
+`projectsToRun` must be modified if a new project is added to `e2e`.
+
+What will happen in order is as following:
+
+1. A bundle (we'll call it `[bundle]`) will be created for `jest-preset-angular` using `npm pack` (`yarn pack` is buggy).
```

Comment:
I'd suggest running from the source, to simplify setup. But running `bundle` has its own benefits, so I won't oppose too much